src/plugins/intel_cpu/tests/functional/specific_tests.cmake

```diff
@@ -52,7 +52,7 @@ endif()
     NAME ${SUBSET_TARGET_NAME}
     ROOT ${CMAKE_CURRENT_SOURCE_DIR}
     INCLUDES ${INCLUDES}
-    EXCLUDED_SOURCE_PATHS ${EXCLUDED_SOURCE_PATHS_FOR_SUBSET_TEST}
+    EXCLUDED_SOURCE_PATHS ${EXCLUDED_SOURCE_PATHS_FOR_SUBSET_TEST} ${EXCLUDED_SOURCE_PATHS}
```

Comment:
Why we need this change?

Reply:
we need this change to fix linkage issue for `ov_cpu_func_tests_subset` target (if `ENABLE_CPU_SUBSET_TESTS_PATH` is defined) on arm platform